lib/types/Rect.lua

```diff
@@ -0,0 +1,70 @@
+local assign = import("../assign")
+local typeof = import("../functions/typeof")
+local Vector2 = import("./Vector2")
+
+local Rect = {}
+
+setmetatable(Rect, {
+	__tostring = function()
+		return "Rect"
```

Comment:
Looks like coverage is missing this line. We should make sure that `typeof` correctly picks up this type which should fix that.